src/Entity/EventGroupCategory.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\EventGroupCategoryRepository")
+ * @ORM\Table(
+ *     name="events_groups_categories",
```

Comment:
you can avoid the name attribute and late doctrine manage it

Reply:
removed